packages/eslint-plugin-next/package.json

```diff
@@ -15,6 +15,10 @@
     "glob": "7.1.7"
   },
   "devDependencies": {
-    "@types/eslint": "7.28.0"
+    "eslint": "7.24.0"
+  },
+  "scripts": {
+    "build": "swc -d lib src",
```

Comment:
While testing this it looks like turbo isn't persisting the `lib` folder in the cache since we only configure the `build` script to look in `dist`

May be alright to to change this to `dist` and it not be a semver issue since that should be considered internals.